frontend/src/Component/Dashboard.jsx

```
@@ -0,0 +1,71 @@
+                        <p className="text-gray-400 flex items-center gap-2 text-lg">
+                            <AcademicCapIcon className="h-5 w-5 text-gray-400" />
+                            {user?.designation}
+                        </p>
+                    </div>
+                </div>
+
+                {/* Download ID Card Button */}
+                <button
+                    onClick={handleDownloadIdCard}
+                    className="flex items-center gap-2 px-4 py-2 mt-4 sm:mt-0 bg-blue-600 hover:bg-blue-700 text-white font-semibold rounded-lg shadow-md transition"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)